lib/src/presentation/widgets/sign_in_text_fields.dart

```diff
@@ -19,22 +20,33 @@ class PasswordTextFormField extends StatelessWidget {
   final TextStyle style;
   final InputDecoration decoration;
 
+  @override
+  _PasswordTextFormFieldState createState() => _PasswordTextFormFieldState();
+}
+
+class _PasswordTextFormFieldState extends State<PasswordTextFormField> {
+  bool obscurePassword = true;
   @override
   Widget build(BuildContext context) {
     final password =
         context.select((SignInHandlerState state) => state.password);
     return TextFormField(
-      style: style,
-      decoration: decoration ??
+      style: widget.style,
+      decoration: widget.decoration ??
           InputDecoration(
             prefixIcon: const Icon(Icons.vpn_key),
             labelText: 'Password',
-            border: OutlineInputBorder(
-              borderRadius: BorderRadius.circular(7),
-            ),
+            suffixIcon: IconButton(
+                icon: obscurePassword
+                    ? FontAwesomeIcons.eye
```

Comment:
Don't want to bring in an additional package just for one icon. Instead can you make changes so that we use `Icons.visibility` and `Icons.visibility_off`. Then you can add ways for people to add their own custom icon if you want? Or leave it as is.